suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -772,6 +783,27 @@ export const selectSelectedDeviceAuthenticity = (state: DeviceRootState) => {
     return device?.id ? deviceAuthenticity?.[device.id] : undefined;
 };
 
+export const selectFailedSecurityChecks = (state: DeviceRootState) => {
+    const device = selectDevice(state);
+
+    return device?.features && device.authenticityChecks !== undefined
```

Comment:
Why is `device?.features` relevant?

Reply:
Ah, thanks for clarification. Maybe introducing a typeguard function for it might make it clearer & reusable, but that's a tiny nit :slightly_smiling_face:

Reply:
I didn't add a type guard, but hopefully made it more readable in 6dbac509081c221c50c51f1fb900e8f1565a16e7.